build-support/bin/generate_docs.py

```diff
@@ -320,6 +320,59 @@ def _link(scope: str, *, sync: bool) -> str:
         url_safe_scope = scope.replace(".", "-")
         return f"reference-{url_safe_scope}" if sync else f"{url_safe_scope}.md"
 
+    @staticmethod
+    def _generate_toml_snippet(option_data, scope) -> str:
```

Comment:
Could we use an actual toml generator here, or is that overkill?

Reply:
I can look at it! I think because the args aren't necessarily valid toml it might need some finagling. For example, `...` is used in a lot of maps so that'd need some.